docs/docsServer.js

```diff
@@ -1,25 +1,40 @@
 const esbuild = require("esbuild");
 const chokidar = require("chokidar");
 const liveServer = require("live-server");
-const { config: devScriptBuildConfig } = require("./buildConfigs");
+const { config: docsBuildConfig } = require("./buildConfigs");
+const { esm: packageBuildConfig } = require("../buildConfigs");
+
+// Update chat package directories since this script is executing in a subdirectory.
+// Also enable sourcemaps for easy debugging in the docs page.
+const normalizedPackageBuildConfig = {
+  ...packageBuildConfig,
+  entryPoints: ["../src/index.tsx", "../src/useChat.ts"],
+  sourcemap: true,
+  outdir: "../lib",
+  outbase: "../src"
+};
 
 (async () => {
+  // Builder for the component package
+  const packageBuilder = await esbuild.context(normalizedPackageBuildConfig);
+
   // Builder for the development page
-  const devPageBuilder = await esbuild.context(devScriptBuildConfig);
+  const docsPageBuilder = await esbuild.context(docsBuildConfig);
 
   chokidar
-    // Watch for changes to dev env code or react-search src
-    .watch(["docs/src/*.{ts,tsx,scss}", "docs/src/**/*.{ts,tsx,scss}", "src/**/*.{ts,tsx,scss}"], {
+    // Watch for changes to docs code or react-search src
```

Comment:
This comment should say "react-chatbot" instead of "reach-search", or you can future-proof it by just saying "widget src".

Reply:
https://github.com/vectara/react-chatbot/pull/9/commits/d439e717267622629e3ea5c4983c52e9dfa59d51